src/components/common/layout/layout.tsx

```diff
@@ -1,20 +1,30 @@
-import React from 'react';
+// import React from 'react';
+import { useRouter } from 'next/router';
 
 import Header from '../header/header';
 import Footer from '../footer/footer';
-import { HomeContainer, Content, HomeWrapper } from './styled';
+import { HomeContainer, Content, HomeWrapper, BackGround } from './styled';
+import SearchBar from '../searchbar/searchBar';
 
 interface HomeLayoutProps {
   children?: React.ReactNode;
 }
+
 const HomeLayout = ({ children }: HomeLayoutProps) => {
+  const router = useRouter();
+
   return (
```

Comment:
router에서 경로만 비교할 경우 `router.pathname`을 써주세요.
9버전 이하에서는`router.route`는 쿼리스트링까지 포함합니다

```suggestion
  const { pathname } = useRouter();

  return (
```

Reply:
아하 ..! 넵 알겠습니다 감사합니다!